tutorials/networking/high_level_multiplayer.rst

```diff
@@ -235,23 +235,34 @@ You might have already noticed something different, which is the usage of the ``
 
     remote func register_player(info):
 
-This keyword has two main uses. The first is to let Godot know that this function can be called from RPC. If no keywords are added,
-Godot will block any attempts to call functions for security. This makes security work a lot easier (so a client can't call a function
-to delete a file on another client's system).
-
-The second use is to specify how the function will be called via RPC. There are four different keywords:
+This keyword is one of many that allow a function to be called by a remote procedure call (RPC). There are six of them total:
 
 - ``remote``
 - ``remotesync``
-- ``master``
 - ``puppet``
+- ``puppetsync``
+- ``master``
+- ``mastersync``
+
+Each of them designate who can call the rpc, and optionally ``sync`` if the RPC can be called locally.
+
+.. note:: If no rpc keywords are added, Godot will block any attempts to call functions remotely.
+          This makes security work a lot easier (so a client can't call a function to delete a file on another client's system).
+
+The ``remote`` keyword can be called by any peer, including the server and all clients.
+The ``puppet`` keyword means a call can be made from the network master to any network puppet.
+The ``master`` keyword means a call can be made from any network puppet to the network master.
```

Comment:
Even after using these keywords a ton, I still need someone to verify that I am describing them correctly here. Also I switched up the ordering to what is probably more likely to be used.